packages/manager/modules/pci/src/projects/project/storages/snapshots/snapshot/create-volume/create-volume.module.js

```diff
@@ -4,9 +4,7 @@ import '@uirouter/angularjs';
 import 'angular-translate';
 import '@ovh-ux/ui-kit';
 import 'ovh-api-services';
-
-import blockStorages from '../../../blocks';
-import volumeEdit from '../../../blocks/block/edit/volume-edit';
+import pciBlockStorage from '@ovh-ux/manager-pci-block-storage';
```

Comment:
not needed as a micro app is independent of other apps

Reply:
https://github.com/ovh/manager/pull/10563#discussion_r1472797974